framework/src/modules/nft/module.ts

```diff
@@ -181,8 +184,24 @@ export class NFTModule extends BaseInteroperableModule {
 		};
 	}
 
-	// eslint-disable-next-line @typescript-eslint/no-empty-function
-	public async init(_args: ModuleInitArgs) {}
+	// eslint-disable-next-line @typescript-eslint/require-await
+	public async init(args: ModuleInitArgs) {
+		const ownChainID = Buffer.from(args.genesisConfig.chainID, 'hex');
+		this._internalMethod.init({ ownChainID });
+		this.method.init({ ownChainID });
+		this.crossChainTransferCommand.init({
```

Comment:
Can initialization for transfer command can also be added here please.
`this._transferCommand.init({ method: this.method, internalMethod: this._internalMethod })`

Reply:
Thanks for noticing this. Updated.